daffodil-propgen/src/main/scala/org/apache/daffodil/propGen/PropertyGenerator.scala

```diff
@@ -762,7 +762,7 @@ import org.apache.daffodil.lib.exceptions.ThrowsSDE
   def getGeneratedFilePath(rootDir: String, pkg: String, filename: String): String = {
     val outDir = new java.io.File(rootDir + "/" + pkg.split('.').reduceLeft(_ + "/" + _))
     outDir.mkdirs()
-    val outPath = s"$outDir/$filename"
+    val outPath = String.valueOf(outDir) + "/" + filename
```

Comment:
This one is not just concatenating two strings, so I think this is better as a string interpolation, particularly as it avoids the String.valueOf call.